File: app/src/main/java/com/github/panda_io/micro_panda/ast/statement/Break.java
package com.github.panda_io.micro_panda.ast.statement;

import com.github.panda_io.micro_panda.ast.Context;

public class Break extends Statement {
    public void validate(Context context) {
        /*
         * if c.LeaveBlock == nil {
         * //TO-DO add check
         * //c.Program.Error(b.Position, "invalid break")
         * }
         */
    }
}

File: MicroPanda/AST/Statement/Break.cs
namespace MicroPanda.AST.Statement;

internal class Break : Statement
{
    internal override void Validate(Context context)
    {
        // TO-DO add check
        // if (context.LeaveBlock == null)
        // {
        //     context.Program.Error(Position, "invalid break");
        // }
    }
}
